lxc/__init__.py

```diff
@@ -119,7 +119,10 @@ def __init__(self, container):
 
     def __getitem__(self, index):
         if index >= len(self):
-            raise IndexError("list index out of range")
+
+            # If index of network is out of bounds, create a new network.
+            print("Network index out of range. Creating new network at index %d." % index)
```

Comment:
Probably don't want that print statement in the final PR :)

Reply:
Just removed it.